server/php/index.php

```diff
@@ -0,0 +1,198 @@
+<?php
+use Slim\Http\Request;
+use Slim\Http\Response;
+use Stripe\Stripe;
+use Store\Inventory;
+use Store\Shipping;
+
+// Due to a bug in the PHP embedded server, URLs containing a dot don't work
+// This will fix the missing variable in that case
+if (PHP_SAPI == 'cli-server') {
+    $_SERVER['SCRIPT_NAME'] = 'index.php';
+}
+
+require __DIR__ . '/vendor/autoload.php';
+
+// Instantiate the app
+$settings = require __DIR__ . '/settings.php';
+$app = new \Slim\App($settings);
+
+// Instantiate the logger as a dependency
+$container = $app->getContainer();
+$container['logger'] = function ($c) {
+  $settings = $c->get('settings')['logger'];
+  $logger = new Monolog\Logger($settings['name']);
+  $logger->pushProcessor(new Monolog\Processor\UidProcessor());
+  $logger->pushHandler(new Monolog\Handler\StreamHandler($settings['path'], $settings['level']));
+  return $logger;
+};
+
+// Middleware
+$app->add(function ($request, $response, $next) {
+  Stripe::setApiKey($this->get('settings')['stripe']['secretKey']);
+  $request = $request->withAttribute('staticDir', $this->get('settings')['stripe']['staticDir']);
+
+	return $next($request, $response);
+});
+
+// Serve the store
+$app->get('/', function (Request $request, Response $response, array $args) {
+  return $response->write(file_get_contents($request->getAttribute('staticDir') . 'index.html'));
+});
+
+// Serve static assets and images for index.html
+$paths = [
+  'javascripts' => 'text/javascript', 'stylesheets' => 'text/css',
+  'images' => FILEINFO_MIME_TYPE,
+  'images/products' => FILEINFO_MIME_TYPE,
+  'images/screenshots' => FILEINFO_MIME_TYPE
+];
+$app->get('/{path:' . implode('|', array_keys($paths)) . '}/{file:[^/]+}',
+  function (Request $request, Response $response, array $args) use ($paths) {
+    $resource = $request->getAttribute('staticDir') . $args['path'] . '/' . $args['file'];
+    if (!is_file($resource)) {
+      $notFoundHandler = $this->get('notFoundHandler');
+      return $notFoundHandler($request, $response);
+    }
+
+    return $response->write(file_get_contents($resource))
+      ->withHeader('Content-Type', $paths[$args['path']]);
+  }
+);
+
+// General config
+$app->get('/config', function (Request $request, Response $response, array $args) {
+  $config = $this->get('settings')['stripe'];
+  return $response->withJson([
+    'stripePublishableKey' => $config['publishableKey'],
+    'stripeCountry' => $config['accountCountry'],
+    'country' => $config['accountCountry'],
```

Comment:
this is meant to be the default country of the payment form, and could therefore be different from your Stripe account's (your business ) country.

Reply:
Settings.php updated.